packages/astro/src/default/pages/index.astro

```diff
@@ -3,10 +3,9 @@ import { getTutorial } from '../utils/content';
 
 const tutorial = await getTutorial();
 
-const parts = Object.values(tutorial.parts);
-const part = parts[0];
-const chapter = part.chapters[1];
-const lesson = chapter.lessons[1];
+const part = tutorial.parts[tutorial.firstPartId!];
+const chapter = part.chapters[part?.firstChapterId!];
+const lesson = chapter.lessons[chapter?.firstLessonId!];
```

Comment:
I'm very confused. So all the `firstPartId` and `firstChapterId` and `firstLessonId` are defined as optional in `entities/index.ts`. Why are we here telling TS that they will be defined?

Also, what if they are defined but they don't exist? Then `chapter` will be undefined and it will throw, no?